src/software/ai/hl/stp/tactic/get_behind_ball/get_behind_ball_fsm.h

```diff
@@ -107,7 +109,7 @@ struct GetBehindBallFSM
             return contains(behind_ball_region, event.common.robot.position()) &&
                    compareAngles(event.common.robot.orientation(),
                                  event.control_params.chick_direction,
-                                 Angle::fromDegrees(5));
+                                 Angle::fromDegrees(15));
```

Comment:
why do we need to increase the tolerance here?

Reply:
I was messing around with this file to get a test to pass, but none of the changes here are necessary, reverted